param/obstacle_avoider.yaml

```diff
@@ -0,0 +1,12 @@
+# How far an obstacle must be from a flight vector for it to be considered safe
+obst_avoid_norm_limit: 1
```

Comment:
Does this mean that you only consider continuing to move forward or turning around when avoiding obstacles? I assume only 1 step in sampling between 0 - 2pi results in just angles of 0 and 2pi off the current velocity vector being considered.

Reply:
As Aaron commented above, this is the farthest perpendicular distance in meters that an obstacle can be to a velocity vector for it to be discarded. I'm updating the comment to make it clearer.

I'm not sure what you mean by continuing forward or turning around - in practice, if the obstacle avoider is given a constant forward velocity vector toward an obstacle, the drone will translate or "slide around" the obstacle.

Reply:
Aaron's comment made the first comment in that file not show up on my screen, so I assumed that the "How many steps to take" comment corresponded to the "obst_avoid_norm_limit" variable. Obviously that makes no sense now, but I thought limit meant the limit on the number of steps